components/Reviews/components/Header/components/AverageRating/index.jsx

```diff
@@ -2,33 +2,52 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import RatingStars from '@shopgate/pwa-ui-shared/RatingStars';
 import RatingCount from 'Components/Reviews/components/RatingCount';
+import Link from '@shopgate/pwa-common/components/Router/components/Link';
+import { ITEM_PATH } from '@shopgate/pwa-common-commerce/product/constants';
+import { bin2hex } from '@shopgate/pwa-common/helpers/data';
 import { container } from './style';
+import connect from './connector';
 
 /**
  * The average rating and number of ratings for a product.
  * @param {Object} rating The rating values.
```

Comment:
please extend the jsDoc here and add your new param. Do it also for the gmd please

Reply:
done